tx_kernel/build.rs

```diff
@@ -0,0 +1,28 @@
+use assembly::{LibraryNamespace, MaslLibrary, Version};
+use std::io;
+
+// CONSTANTS
+// ================================================================================================
+const ASM_DIR_PATH: &str = "./asm";
+const ASL_DIR_PATH: &str = "./assets";
+
+// PRE-PROCESSING
+// ================================================================================================
+
+/// Read and parse the contents from `./asm` into a `LibraryContents` struct, serializing it into
+/// `assets` folder under `std` namespace.
+#[cfg(not(feature = "docs-rs"))]
+fn main() -> io::Result<()> {
```

Comment:
I removed the documentation generation since there is not much going. Let me know if I should add back (and were should we put the md generator)